### PR TITLE
docs: add changelog for 4.0.0

### PR DESCRIPTION
**Pull request by vincentdelmundo**

### CODE REVIEW

File: inappmessaging/USERGUIDE.md [hunk 1]
```diff
@@ -312,6 +312,13 @@ Documents targeting Product Managers:
 
 ## <a name="changelog"></a> Changelog
 
+### 4.0.0 (in-progress)
+* SDKCF-3651: Change config request to POST
```
Comment (donnie-jp) [inline inappmessaging/USERGUIDE.md:316]: Not POST -> GET?

File: inappmessaging/USERGUIDE.md [hunk 2]
```diff
@@ -312,6 +312,13 @@ Documents targeting Product Managers:
 
 ## <a name="changelog"></a> Changelog
 
+### 4.0.0 (in-progress)
```
Comment (donnie-jp) [inline inappmessaging/USERGUIDE.md:315]: Version should be 4.0.0-pp-v1

**Review by donnie-jp — CHANGES_REQUESTED** [review]
See inline comments

File: inappmessaging/USERGUIDE.md (hunk 1 above)
Comment (vincentdelmundo) [reply inappmessaging/USERGUIDE.md]: updated

File: inappmessaging/USERGUIDE.md (hunk 2 above)
Comment (vincentdelmundo) [reply inappmessaging/USERGUIDE.md]: updated.
is the "-v1" necessary, since we use semver?

**Review by imaginaris — APPROVED** [review]

File: inappmessaging/USERGUIDE.md (hunk 2 above)
Comment (vincentdelmundo) [reply inappmessaging/USERGUIDE.md]: changed to "4.0.0-pp"

**Review by donnie-jp — APPROVED** [review]
LGTM

**Pull request merged**